src/web_player.js

```diff
@@ -1018,6 +1029,8 @@ export class WebPlayer {
         this.setPlayerVisible(false);
 
         this.containerElement.innerHTML = this.videoHTMLContent;
+        this.isIpCameraBroadcast()
```

Comment:
No need to call this method if `restJwt` is blank

Reply:
Done
Thanks for the review